pages/science/flames.rst

```diff
@@ -125,6 +125,40 @@ where :math:`D_{ki}` is the multicomponent diffusion coefficient and
 :math:`D_k^T` is the Soret diffusion coefficient (used only if calculation of
 this term is specifically enabled).
 
+The Drift-Diffusion Model
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To account for the transport of charged species in a flame, the drift term is added to the
+diffusive fluxes of the mixture-average formulation according to [Ped1993]_,
+
+.. math::
+  
+   j_k^* = \rho \frac{W_k}{\overline{W}} D_{km}^\prime \frac{\partial X_k}{\partial z} +
+           s_k \mu_k E Y_k,
+
+where :math:`s_k` is the sign of charge (1,-1, and 0 respectively for positive, negative, and
+neutral charge), :math:`\mu_k` is the mobility, and :math:`E` is the electric field.
+The diffusion coefficients and mobilities of charged species can be more accurately calculated by
+**IonGasTransport::getMixDiffCoeffs** and **IonGasTransport::getMobilities**. In addition, the
+correction can only be applied to the fluxes of neutral species to ensure the accuracy of the
+fluxed of charged species,
```

Comment:
The wording of this is a bit confusing, and I have a couple questions about this. First, I'd try something more like
> In addition, the following correction is applied to the fluxes of neutral species only, to preserve the correct fluxes of charged species:
But beyond that, I'm wondering, does this correction still preserve the condition that the sum of the diffusive mass fluxes is zero?

Reply:
Yes, it only corrects the flux of neutral species to ensure zero diffusive flux.